src/examples/libpmemobj/tree_map/data_store.c

```diff
@@ -94,13 +94,19 @@ dec_keys(uint64_t key, PMEMoid value, void *arg)
 	return 0;
 }
 
-int main(int argc, const char *argv[]) {
+int
+main(int argc, const char *argv[])
+{
 	if (argc < 2) {
-		printf("usage: %s file-name\n", argv[0]);
+		printf("usage: %s file-name [nops]\n", argv[0]);
 		return 1;
 	}
 
 	const char *path = argv[1];
+	int nops = MAX_INSERTS;
```

Comment:
This will break for any nops larger than MAX_INSERTS. Please allocate the 'keys' array dynamically...